src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx

```diff
@@ -5,7 +5,7 @@ export const meta = {
   title: 'Connect to data from server-side runtimes',
   description:
     'Learn more about how to call an AppSync GraphQL API from a NodeJS app or a Lambda function',
-  platforms: ['javascript', 'nextjs', 'vue', 'angular', 'react', 'react-native', 'android', 'swift']
```

Comment:
This is as expected because there are contents below that are applicable to customers using android & iOS but with Lambda functions.

Reply:
Ack reverted this back for the v6 page. @katiegoines just note this update I was wrong to remove these two so they should stay on this page.